book/src/configuration.md

```diff
@@ -332,3 +334,11 @@ max-wrap = 25 # increase value to reduce forced mid-word wrapping
 max-indent-retain = 0
 wrap-indicator = ""  # set wrap-indicator to "" to hide it
 ```
+
+### `[editor.explorer]` Section
+Sets explorer side width and style.
+
+ | Key            | Description                                                                              | Default |
+ | ---            | -----------                                                                              | ------- |
+ | `column-width` | explorer side width                                                                      | 30      |
+ | `position`     | explorer widget position, `overlay`, `right`, or `left`                                  | `left`  |
```

Comment:
```suggestion
 | `position`     | explorer widget position, `overlay`, `left`, or `right`                                  | `left`  |
```
Since english it is usually from left to right I think it is better to arrange it this way.